SD_ROOT/wz_mini/wz_mini.conf

```diff
@@ -96,4 +82,4 @@ DEBUG_INITRAMFS_ENABLED="false"
 DEBUG_PASSWORD="false"
 
 #####SCRIPTING#####
-CUSTOM_SCRIPT_PATH=""
+CUSTOM_SCRIPT_PATH="/opt/wz_mini/etc/crontab/start_cron.sh"
```

Comment:
The cron startup probably shouldn't be using the custom script hook, since users can't then use it for other things anymore.

Reply:
Don't mind moving it, though im not sure where I could put it that would gaurantee this script (which starts up cron mainly, or the cron command itself) could be moved to that wouldnt interfere? Could I make something in the conf file that points to a script to be run? Im not super familiar with the startup sequence of this camera....Though im open to a suggestion.